Fix #2740. Use chrome instead of phantomjs for testing.

e-carlin opened the pull request:
Phantomjs doesn't support ES6 and it is deprecated.
Use Chrome Headless for testing instead.

This depends on radiasoft/download#115.

e-carlin commented:
Yes https://github.com/radiasoft/sirepo/pull/2745/commits/51774c2ded4dccf7b7faf294a97e94f55bfe94fb

I think adding it to beamsim-codes make sense.

robnagler commented:
It's about 100MB, because it brings in wayland... This is how we get to 10GB images. Go ahead and fix it including test.sh

e-carlin commented:
I added chrome to common https://github.com/radiasoft/download/pull/115/commits/264145987744952a5172b3927748c89f1236e24d

I also removed es6 checks from test.sh https://github.com/radiasoft/sirepo/pull/2745/commits/0f03d29a5664b789d5f1a7c6bb79c740f4d78fe7
